distributed-coordinator/README.md

```diff
@@ -1,7 +1,3 @@
 # Gentics Mesh - Distributed Coordinator
 
-This module contains the clustering code for the coordination feature / request delegation and needed master election.
-
-## MDM
-
-Must be made independent from OrientDB.
+This module contains the clustering code for the coordination feature / request delegation and needed master election. OrientDB-specific implementation.
```

Comment:
The coordination feature, especially the master election is not OrientDB specific.
The elected master will also be the instance, which runs background jobs (like schema migration) and handles elasticsearch sync